datadog/resource_datadog_synthetics_test_.go

```diff
@@ -49,6 +49,28 @@ func resourceDatadogSyntheticsTest() *schema.Resource {
 				Type:     schema.TypeMap,
 				Optional: true,
 			},
+			"request_query": {
+				Type:     schema.TypeMap,
+				Optional: true,
+			},
+			"request_basicauth": {
```

Comment:
For my knowledge, you said `As request_headers, this needs to be outside of the request` but why?